.github/workflows/mutation.yml

```diff
@@ -78,7 +77,9 @@ jobs:
         path: mutation/ingest
 
     - name: Ingest
-      run: perl regen_index.pl >> $GITHUB_STEP_SUMMARY
+      run: perl regen_index.pl $GITHUB_HEAD_REF >> $GITHUB_STEP_SUMMARY
```

Comment:
You can emit markdown in to the `$GITHUB_STEP_SUMMARY` variable and it gets displayed in the build summary, [for example](https://github.com/Mastercard/flow/actions/runs/4477224495#summary-12154525627)